From 937f06c25a92968a82428d8e502ac42a3e91382c Mon Sep 17 00:00:00 2001
From: Herton Ronaldo Krzesinski <herton@mandriva.com.br>
Date: Tue, 23 Feb 2010 14:29:54 -0300
Subject: [PATCH 5/8] Fix sorting of malayalam letter 'na'.

Fix sorting of malayalam letter 'na'.
Add missing bz number.

(cherry-picks from 1e5e9ec825bb67705b39460826e80750ae1ae379,
1a36db1c125785826afbf169d26a9afd98577d65)
---
 localedata/ChangeLog                  |    6 ++++++
 localedata/locales/iso14651_t1_common |    4 ++--
 2 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/localedata/ChangeLog b/localedata/ChangeLog
index e2f26de..e88cbd6 100644
--- a/localedata/ChangeLog
+++ b/localedata/ChangeLog
@@ -1,3 +1,9 @@
+2010-02-03  Ulrich Drepper  <drepper@redhat.com>
+
+	[BZ #10414]
+	* locales/iso14651_t1_common: Fix sorting of malayalam letter 'na'.
+	Patch by Pravin Satpute <psatpute@redhat.com>.
+
 2010-01-25  Andreas Schwab  <schwab@redhat.com>
 
 	* locales/i18n: Don't map U00DF to U1E9E in toupper table.
diff --git a/localedata/locales/iso14651_t1_common b/localedata/locales/iso14651_t1_common
index edbe9e0..a7e62d3 100644
--- a/localedata/locales/iso14651_t1_common
+++ b/localedata/locales/iso14651_t1_common
@@ -6272,8 +6272,8 @@ order_start <MALAYALAM>;forward;forward;forward;forward,position
 <U0D26>	"<ml-dda><ml-virama><mlvw-shorta>";<BAS>;<MIN>;IGNORE
 <ml-ddh> "<ml-ddha><ml-virama>";<BAS>;<MIN>;IGNORE
 <U0D27>	"<ml-ddha><ml-virama><mlvw-shorta>";<BAS>;<MIN>;IGNORE
-<ml-chillu-n> "<ml-chillu-n>";<BAS>;<MIN>;IGNORE # ന്‍= ന  + ്  + zwj
-<U0D28>	"<ml-chillu-n><ml-virama><mlvw-shorta>";<BAS>;<MIN>;IGNORE  #ന = ന + ് + അ
+<ml-chillu-n> "<ml-na><ml-virama>";<BAS>;<MIN>;IGNORE # ന്‍= ന  + ്  + zwj
+<U0D28>	"<ml-na><ml-virama><mlvw-shorta>";<BAS>;<MIN>;IGNORE  #ന = ന + ് + അ
 <ml-p>	"<ml-pa><ml-virama>";<BAS>;<MIN>;IGNORE
 <U0D2A>	"<ml-pa><ml-virama><mlvw-shorta>";<BAS>;<MIN>;IGNORE
 <ml-ph>	"<ml-pha><ml-virama>";<BAS>;<MIN>;IGNORE
-- 
1.7.0

